pkgs/development/python-modules/pyaxmlparser/default.nix

```diff
@@ -1,16 +1,31 @@
-{ buildPythonPackage, stdenv, lxml, click, fetchPypi }:
+{ buildPythonPackage, stdenv, lxml, click, fetchFromGitHub, pytest, isPy3k }:
 
 buildPythonPackage rec {
   version = "0.3.13";
   pname = "pyaxmlparser";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1mzdrifnaky57vkmdvg0rgjss55xkxaramci3wpv4h65lmk95988";
+  # the PyPI tarball doesn't ship tests.
+  src = fetchFromGitHub {
+    owner = "appknox";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0jfjhxc6b57npsidknxmhj1x813scg47aaw90ybyr90fpdz5rlwk";
   };
 
+  # runtime fails with `TypeError: AXMLParser object is not an iterator`
+  disabled = !isPy3k;
```

Comment:
The project page also states that this is a python3 project: https://github.com/appknox/pyaxmlparser